src/defaults.js

```diff
@@ -364,6 +364,12 @@ module.exports = {
      * @instance
      */
     fixedColAlign: 'center',
+    /**
+     * @default
+     * @type {string}
+     * @instance
+     */
+    colHAlign: 'center',
```

Comment:
Remove "col" because it can also pertain the the grid as a whole. Just make it 'halign'.